weave/trace/custom_objs.py

```diff
@@ -51,8 +57,12 @@ def decode_custom_obj(
     encoded_path_contents: Mapping[str, Union[str, bytes]],
     load_instance_op_uri: Optional[str],
 ) -> Any:
-    load_instance_op = None
-    if load_instance_op_uri is not None:
+    if weave_type["type"] in KNOWN_TYPES:
+        serializer = get_serializer_by_id(weave_type["type"])
```

Comment:
what happens if the serialization changes? It seems like we should first "try" the local version, then fallback to the remote version, then error

Reply:
ok yeah that's safer